pykechain/client.py

```diff
@@ -1556,6 +1556,43 @@ def _create_parts_bulk(
         parts = self.parts(id__in=",".join(part_ids))
         return PartSet(parts=parts)
 
+    def _delete_parts_bulk(
+            self,
+            parts: List[Union[Part2]],
```

Comment:
The type hinting does not agree with the `List[Property] or List[UUID]` support. On that item, is it necessary to support both? At least should be all the same type in a list?

Reply:
List[Property] is not necessary as we're only dealing with parts. List[UUID] is indeed accepted so I will make sure that is adapted.